index.html

```diff
@@ -16,24 +16,35 @@
         <ul class="header-nav">
           <li><a class="header-nav-link" href="#">Home</a></li>
           <li><a class="header-nav-link" href="./pages/about.html">About Us</a></li>
-          <li><a class="header-nav-link" href="#">Our Course</a></li>
+          <li><a class="header-nav-link" href="./pages/course.html">Our Course</a></li>
           <li><a class="header-nav-link" href="#">Link four</a></li>
         </ul>
       </nav>
       </div>
     </header>
     <main>
       <!--1st section - Hero -->
-      <section class="hero">
-        <div class="container">
+      <section class="section hero">
+        <div class="container hero-container">
           <h1 class="hero-title">This website is awesome</h1>
           <p class="hero-text">
             Lorem ipsum dolor sit amet consectetur adipisicing elit. 
             Quisquam, voluptatum. Quisquam, voluptatum. Quisquam, voluptatum.
           </p>
           <a class="hero-btn" href="#">Signup</a>
         </div>
+
+          <div class="image-container">
+            <img 
+              class="hero-img" 
+              src="./images/heroimg.jpg" 
+              alt="a person sitting in front of a computer"
```

Comment:
I would probably write alt="an open laptop with code display on a desk with a plant and a mug" for the hero img